assets/js/app/Data.js

```diff
@@ -235,7 +235,17 @@ LocusZoom.Data.Source.prototype.parseArraysToObjects = function(x, fields, outna
     fields.forEach(function(f, i) {
         if (!(f in x)) {throw "field " + f + " not found in response for " + outnames[i];}
     });
-    var N = x[Object.keys(x)[1]].length;
+    var x_keys = Object.keys(x);
+    var N = x[x_keys[0]].length; // NOTE: this used 1 instead of 0 before, why?
```

Comment:
Looking at the type of responses that come back where this method is invoked using 1 instead of 0 seems arbitrary. It could be that the endpoint originally gave back an object where the first key was not an array of length consistent with the other keys, but that no longer appears to be the case. It's certainly less safe to assume the object coming back will have at least two keys, so I think you're in the right to look at the 0th key here and can safely drop this comment.